spec/01-unit/011-balancer_spec.lua

```diff
@@ -506,6 +506,15 @@ describe("Balancer", function()
       })
       assert.are.same(crc32(value), hash)
     end)
+    it("cookie", function()
+      local value = "some cookie value"
+      ngx.var.cookie_Foo = value
+      local hash = balancer._create_hash({
+          hash_on = "cookie",
+          hash_on_cookie = "Foo",
+      })
+      assert.are.same(crc32(value), hash)
```

Comment:
can we also add validation of the `ctx` variables?, so in this case that `ngx.ctx.SET_HASH_ON_COOKIE` is NOT created.

And add another test without a cookie set, that shows that `ngx.ctx.SET_HASH_ON_COOKIE` is properly created?